cloudinit/config/cc_growpart.py

```diff
@@ -208,13 +259,18 @@ def get_size(filename):
         os.close(fd)
 
 
-def device_part_info(devpath):
+def device_part_info(devpath, is_lvm):
     # convert an entry in /dev/ to parent disk and partition number
 
     # input of /dev/vdb or /dev/disk/by-label/foo
     # rpath is hopefully a real-ish path in /dev (vda, sdb..)
     rpath = os.path.realpath(devpath)
 
+    # first check if this is an LVM and get its PVs
+    lvm_rpath = get_pvs_for_lv(devpath)
```

Comment:
Won't this call raise an exception on any system that doesn't have "lvm" command installed?

Reply:
yeah... we shouldnt stack trace on that. we should log info (or less) and not fail.